rfc8826.xml

```diff
@@ -22,60 +24,74 @@
-        Web conferencing, and direct data transfer. Unlike most conventional real-time systems,
-        (e.g., SIP-based <xref target="RFC3261" format="default"/> soft phones) WebRTC communications are directly controlled
-        by some Web server. A simple case is shown below.
+        Web conferencing, and direct data transfer. Unlike most conventional real-time systems
+        (e.g., SIP-based <xref target="RFC3261" format="default"/> soft
+	phones), WebRTC communications are directly controlled by some Web
+	server. A simple case is shown below. 
       </t>
       <figure anchor="fig.simple">
-        <name>A simple WebRTC system</name>
+        <name>A Simple WebRTC System</name>
         <artwork name="" type="" align="left" alt=""><![CDATA[
-                            +----------------+
-                            |                |
-                            |   Web Server   |
-                            |                |
-                            +----------------+
-                                ^        ^
-                               /          \
-                       HTTP   /            \   HTTP
-                        or   /              \   or
-                 WebSockets /                \ WebSockets
-                           v                  v
-                        JS API              JS API
-                  +-----------+            +-----------+
-                  |           |    Media   |           |
-                  |  Browser  |<---------->|  Browser  |
-                  |           |            |           |
-                  +-----------+            +-----------+
-                      Alice                     Bob ]]></artwork>
+                          +----------------+
+                          |                |
+                          |   Web Server   |
+                          |                |
+                          +----------------+
+                              ^        ^
+                             /          \
+                     HTTP   /            \   HTTP
+                      or   /              \   or
+               WebSockets /                \ WebSockets
+                         v                  v
+                      JS API              JS API
+                +-----------+            +-----------+
+                |           |    Media   |           |
+                |  Browser  |<---------->|  Browser  |
+                |           |            |           |
+                +-----------+            +-----------+
+                    Alice                     Bob ]]></artwork>
       </figure>
       <t>
         In the system shown in <xref target="fig.simple" format="default"/>, Alice and Bob both have
-        WebRTC-enabled browsers and they visit some Web server which operates a
+        WebRTC-enabled browsers and they visit some Web server that operates a
```

Comment:
Please revert all that/which changes. See: 
http://itre.cis.upenn.edu/%7Emyl/languagelog/archives/001464.html